DB/src/main/java/io/deephaven/db/util/PythonScopeJpyImpl.java

```diff
@@ -1,17 +1,27 @@
 package io.deephaven.db.util;
 
 import org.jpy.PyDictWrapper;
+import org.jpy.PyModule;
 import org.jpy.PyObject;
 
+import java.util.*;
 import java.util.Map.Entry;
-import java.util.Optional;
 import java.util.stream.Stream;
 
 public class PythonScopeJpyImpl implements PythonScope<PyObject> {
     private final PyDictWrapper dict;
+    private static PyObject NumbaVectorizedFuncType;
+    {
+        try {
+            NumbaVectorizedFuncType = PyModule.importModule("numba.np.ufunc.dufunc").getAttribute("DUFunc");
+        } catch (Exception e) {
+            NumbaVectorizedFuncType = null;
+        }
+    }
```

Comment:
I am a little worried about the static nature of this - for some of the unit tests, we start and stop python interpreter - I would make a note here that we may have to re-investigate this specific code in the future.